gossip/src/cluster_info.rs

```diff
@@ -765,7 +765,7 @@ impl ClusterInfo {
                         ""
                     },
                     now.saturating_sub(last_updated),
-                    node.pubkey(),
+                    node.pubkey().to_string(),
```

Comment:
Neat. Just curious, can we use {:<44} in the format string instead of to_string() here?

Reply:
doesn't change the output

Reply:
I see, I think it's fine to use to_string() instead of using the pubkey formatting.